crates/dojo-lang/src/contract.rs

```diff
@@ -48,6 +48,8 @@ impl DojoContract {
                     use dojo::world;
                     use dojo::world::IWorldDispatcher;
                     use dojo::world::IWorldDispatcherTrait;
+                    use dojo::upgradable::{IUpgradeable, UpgradeableTrait};
+                    use starknet::ClassHash;
```

Comment:
use fully qualified calls in the impl to avoid import collisions with the users contract (so remove these imports)